src/lesson13/CollectionsTraining.java

```diff
@@ -0,0 +1,92 @@
+package lesson13;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CollectionsTraining {
+
+
+    /**
+     * Task 1. Method counts occurrence of string to find in list.
+     *
+     * @param strings      list of strings used to find a target
+     * @param targetString for which it is necessary to count occurrence
+     * @return amount of target string occurrence in list
+     */
+    public int countOccurrence(List<String> strings, String targetString) {
+        int count = 0;
+        for (String string : strings) {
+            if (string.equals(targetString)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    /**
+     * Task 2. Method that converts array to list
+     *
+     * @param values array of ints
+     * @return list of ints from array
+     */
+    public List<Integer> toList(int[] values) {
+        ArrayList<Integer> integers = new ArrayList<>();
```

Comment:
Надо работать с абстракцией, а не с реализацией. Иначе это убивает возможность подстановки (Liskov Substituion)
```suggestion
        List<Integer> integers = new ArrayList<>();
```